infra/outlines.py

```diff
@@ -57,21 +57,6 @@ def html(self):
     def sub(self):
         return None
 
-@dataclass
-class IfMain:
```

Comment:
I guess this was dead code?

Reply:
Yes—we used to include the `if __name__` block in the outline but later we refactored to have a `mainloop` function and I decided that listing it added little-to-no value.